src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx

```diff
@@ -21,10 +37,28 @@ export const AUCTION_ARTWORKS_IMAGE_HEIGHT = 160
 
 export const AuctionArtworksRail: React.FC<AuctionArtworksRailProps> = ({
   sale,
+  tabType,
   ...rest
 }) => {
   const ref = useRef<HTMLDivElement | null>(null)
+  const { trackEvent } = useTracking()
   const { isEnteredView, Waypoint } = useLazyLoadComponent()
+  const { contextPageOwnerType } = useAnalyticsContext()
+  const contextModule = getContextModule(tabType)
+
+  const trackViewSaleClick = () => {
+    trackEvent(
+      clickedArtworkGroupHeader({
```

Comment:
For lots and lots of common interactions there's helper functions that one can import and fill out; here's an example of possible [click helpers](https://github.com/artsy/cohesion/tree/master/src/Helpers/Click), though there's lots of stuff in that parent `/Helpers` directory.